docs/_articles/en/user-guide/remote-development.md

```diff
@@ -30,32 +30,32 @@ To start remote development in dev container, install:
 - Latest version of the [Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode)
 
-## Try a Dev Container with a Sample Project
-
-Let’s use a sample project we provide and try remote development in Docker container with Salesforce Extension.
-
-1. Clone the [sample repository](https://github.com/forcedotcom/vscode-remote-try-sfdx). The `.devcontainer` folder in the sample project contains the `devcontainer.json` file that defines how to configure the dev container, the Dockerfile to use, and the extensions to install. For details, see [Creating a devcontainer.json file](https://code.visualstudio.com/docs/remote/containers#_creating-a-devcontainerjson-file) and [devcontainer.json reference](https://code.visualstudio.com/docs/remote/containers#_devcontainerjson-reference).
-1. Run `Remote-Containers: Open Folder in Container` from the Command Palette and open the project you cloned. VS code creates a dev container the first time you open the project. After the dev container is built, the project folder in your local system automatically connects and maps into the container, and the side bar shows **Dev Container: Salesforce Project**. The container will pre-install and configure Java, Git, Salesforce CLI, and all other extensions defined in the `devcontainer.json` file.
-
-   ![Dev Coontainer](./images/devcontainer.png)
-
-1. From the Command Palette, run the `SFDX:Authorize a Dev Hub` in the container. From the Output panel (below the editor region), you can get the user code and the verification URL required to complete the authorization.
-   If you are logged in to an org that you don’t want to authorize as a Dev Hub, make sure you log out. Otherwise you’ll not be prompted to enter the credentials for the org you want to authorize.
-   After the authorization is complete, the message `SFDX: Authorize a Dev Hub successfully ran` is displayed. In case the success message is not displayed, check if you have the correct user code and try again. If the login page doesn’t prompt you to re-enter the code, terminate the command and run it again.
-   If you are interested, read more about [Device Authentication Flow](https://help.salesforce.com/articleView?id=remoteaccess_oauth_device_flow.htm&type=5).
-   ![Authorize Success Message](./images/authorize_message.png)
-1. Run `SFDX: Create a Default Scratch Org`.
-
-   You’re all set to use the container as a full-featured development environment.
-
 ## Open a Project in a Dev Container
 
 1. Open an existing project you want to work with or create a new project.
-1. From the Command Palette, run `Remote-Container: Add Development Container Configuration Files`.
+1. From the Command Palette, run **Remote-Containers: Add Development Container Configuration Files**.
```

Comment:
Apologies! Not my change.

Reply:
@jag-sfdc - I updated this article last week - I fixed several instances of incorrect tagging. When you select a command, it should be bold. File names and output should be in mono-font.

Reply:
@jag-sfdc @lcampos - do you know why this article update is showing up here? Maybe I didn't merge it yet?

Reply:
@ekapner If Jag was able to pull it, it's definitely been merged in. It sounds like Jag just needs to rebase. Right now the history thinks that your change is part of his PR. After rebasing it should no longer be listed as a commit in this PR.